### PR TITLE
feat: github actions ci

### PR DESCRIPTION
**Pull request by f0rmiga**
After discussing with @carolynvs, we decided that we will replace the now-dead Travis with GitHub Actions.

### CODE REVIEW

File: cmd/minibroker/main.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func runWithContext(ctx context.Context) error {
 }
 
 func cancelOnInterrupt(ctx context.Context, f context.CancelFunc) {
-	term := make(chan os.Signal)
+	term := make(chan os.Signal, 1)
```
Comment (mook-as) [inline cmd/minibroker/main.go:157]: Thought `make(chan …)` defaulted to 1. Doesn't hurt either way, at least.

**Review by mook-as — APPROVED** [review]
/lgtm

File: cmd/minibroker/main.go (hunk 1 above)
Comment (f0rmiga) [reply cmd/minibroker/main.go:157]: Not true. The former was an unbuffered channel. I believe this change was `go vet` yelling at me.

**Pull request merged**